CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ CHANGELOG
 **BUG FIXES**
 - Fix default for disable validate and test components when building custom AMI. The default was to disable those components, but it wasn't effective.
 - Handle corner case in the scaling logic when instance is just launched and the describe instances API doesn't report yet all the EC2 info.
+- Fix validation for configuration parameter `Scheduling/SlurmQueues/ComputeResources/DisableSimultaneousMultithreading` 
```

Comment:
isn't the DisableSimultaneousMultithreading fixed also for the HeadNode?

Reply:
Good point, thanks! It is. I would put a general 

"Fix validation for configuration parameter `DisableSimultaneousMultithreading` to allow the disablement of multi-threading on all instances supporting it." then